client/containers/workflow-list/component.vue

```diff
@@ -207,28 +208,45 @@ export default {
         return { workflows, nextPageToken };
       }
 
-      this.loading = true;
-      this.error = undefined;
-
       const includeStatus = ![STATUS_ALL, STATUS_OPEN, STATUS_CLOSED].includes(
         queryWithStatus.status
       );
       const { status, ...queryWithoutStatus } = queryWithStatus;
       const query = includeStatus ? queryWithStatus : queryWithoutStatus;
 
       try {
-        const res = await httpService.get(url, { query });
+        if (this.abortController) {
+          this.abortController.abort();
+          await delay();
```

Comment:
This `delay` is needed here because otherwise the loading spinner in the other thread will not get reset correctly and will end up not showing a spinner for the new request.